dace/transformation/pass_pipeline.py

```diff
@@ -22,14 +22,15 @@ class Modifies(Flag):
     Symbols = auto()  #: Symbols were modified
     States = auto()  #: The number of SDFG states and their connectivity (not their contents) were modified
     InterstateEdges = auto()  #: Contents (conditions/assignments) or existence of inter-state edges were modified
+    CFG = States | InterstateEdges #: A CFG (any level) was modified (connectivity or number of control flow blocks, but not their contents)
```

Comment:
I'd rather you place any non-`auto()` value after all the autos, so that it's easier to count.